### ISSUE TITLE
Search For Package - add ability to start search with the enter or return button

### ISSUE THREAD

**Issue by shoulders** [webmin/webmin#2089]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.3     |
|  Webmin version      |     2.105      |
|  Usermin version     |     2.005     |
|  Virtualmin version  |     7.9.0       |
|  Theme version       |     21.09.5      |
|  Package updates     |      6  package updates are available   |

### The Background
on the page `Webmin --> System --> Software Packages` there is a Search For Package` feature

![image](https://github.com/webmin/webmin/assets/319997/e98ad4f4-3353-4287-a02f-d3b0fdcbfeea)

### The issue
When you type a search term such as `mysql` and press the **return** or **enter** button nothing happens. All of the other input boxes I have tried respond to these buttons.

### Solution
Allow the `Search For Package` form to be submitted by pressing the **return** or **enter** button.

**Comment by iliajie** [comment]
Hello,

Thanks for the heads up! The fix for this issue is here https://github.com/webmin/webmin/commit/cf396e6ec16b3ccd212b284a08e4d9764ae8892d. The problem was that the old Framed Theme is much more forgiving to invalid HTML.

Jamie, we need to rework this page to use UI lib, if possible?

**Comment by jcameron** [comment]
Yes, using standard ui-lib elements sounds good. I cleaned some of them up ...

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run. @shoulders, sorry for the noise, and thanks for the contribution. We definitely didn't mean any disrespect.

If it turns out the issue is already resolved for you, you're welcome to close it.
